File: Hiring.Challenge.Domain/Models/Repository.cs
using System;

namespace Hiring.Challenge.Domain.Models
{
    public class Repository
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string Name { get; set; }
        public string FullName { get; set; }
        public User Owner { get; set; }
        public string Description { get; set; }
        public string Url { get; set; }
        public string Language { get; set; }
        public int Stars { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime LastUpdate { get; set; }
    }
}

File: br/com/devhiringchallenge/backend/src/models/repository.js
class Repository {
  constructor(id, name, full_name, url, stars, description, language, user_id) {
    this.id = id;
    this.name = name;
    this.full_name = full_name;
    this.url = url;
    this.stars = stars;
    this.description = description;
    this.language = language;
    this.user_id = user_id;
  }
}

module.exports = { Repository }